packages/core/src/components/tabs/tabs.tsx

```diff
@@ -141,15 +158,70 @@ export class Tabs extends AbstractPureComponent<ITabsProps, ITabsState> {
                     role="tablist"
                 >
                     {tabIndicator}
-                    {tabTitles}
+                    {this.props.overflow ? (
+                        <OverflowList
+                            // @ts-ignore
+                            items={tabTitles}
+                            collapseFrom={"start"}
+                            observeParents={true}
+                            overflowRenderer={this.overflowRenderer}
+                            visibleItemRenderer={this.visibleItemsRenderer}
+                            ref={this.refHandlers.overflowList}
+                            {...this.props.overflowListProps}
+                        />
+                    ) : (
+                        tabTitles
+                    )}
                 </div>
                 {tabPanels}
             </div>
         );
     }
 
+    public overflowRenderer(items: Array<React.ReactElement<ITabProps>>) {
+        if (this.props.overflowListProps.overflowRenderer) {
+            return this.props.overflowListProps.items;
+        }
+
+        const overflowMenu = (
+            <Menu>
+                {items.map(item => {
+                    if (item.props.id) {
+                        const onClick = (e: React.MouseEvent<HTMLElement>) => {
+                            item.props.onClick(item.props.id, e);
+                        };
+
+                        return (
+                            <MenuItem
+                                text={item.props.title}
+                                onClick={onClick}
+                                disabled={item.props.disabled}
+                                active={item.props.selected}
+                            />
+                        );
+                    } else {
+                        return item;
+                    }
+                })}
+            </Menu>
+        );
+
+        return (
+            <Popover content={overflowMenu}>
+                <span style={{ lineHeight: "30px", cursor: "pointer" }}>Overflow</span>
```

Comment:
gonna need to customize this.

Reply:
it's passed in as props, I figured there should be a default one which could be overwritten but we can just move it into the examples instead, they are both required props.